test/built-ins/TypedArray/from/length.js

```diff
@@ -2,7 +2,7 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-es6id: 22.2.2.1
+id: sec-%typedarray%.from
```

Comment:
it was necessary as they changed.

es6id is still valid and up to date if the tests are in conformance with released spec without changes on the draft or any stage 3 feat.